util/test/annotate.py

```diff
@@ -45,10 +45,10 @@ def load(path):
   return data
 
 
-def get(data, graph, series, start, end):
+def get(data, graph, series, start, end, hostname=_hostname):
```

Comment:
I think I'd just call it `name` or `config_name`. Right now we only call this code from `genGraphs` and I don't think that's likely to change.